### PR TITLE
Newpulldown_location-rohit.jason

### PR DESCRIPTION
**Pull request by rohitdanda**
Location Longitude and latitude was pull request close

### CODE REVIEW

File: _pins/rohit-rohitdanda.jason [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: rohitdanda
```
Comment (rohitdanda) [inline _pins/rohit-rohitdanda.jason]: great piece of work

**Comment by rohitdanda** [comment]
@rohitdanda tada

File: _pins/rohit-rohitdanda.jason [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+latitude: 53.354
+longitude: -6.252
+Andriod Os was updated
```
Comment (rohitdanda) [inline _pins/rohit-rohitdanda.jason]: new line added

**Comment by JDRLinux** [comment]
Way to go!!!

File: tests/test_fileEndingsPins.rb [hunk 3]
```diff
@@ -5,7 +5,7 @@ class TestFileTypePins < Test::Unit::TestCase
   def test_filecount
     Dir.foreach('./_pins/') do |item|
       next if item == '.' or item == '..'
-        file_exist = File.fnmatch?('*.json', item.downcase)
+        file_exist = File.fnmatch?('Newpulldown_location-rohit.json', item.downcase)
```
Comment (hectorsector) [inline tests/test_fileEndingsPins.rb]: @rohitdanda you'll need to remove this change -- this file houses the automated tests and shouldn't be touched.

File: _pins/rohit-rohitdanda.json [hunk 4]
```diff
@@ -0,0 +1,6 @@
+---
+githubHandle: rohitdanda
+latitude: 53.354
+longitude: -6.252
+Andriod Os was updated
```
Comment (hectorsector) [inline _pins/rohit-rohitdanda.json]: @rohitdanda remove this line -- the automated tests are looking for a very specific set of words to make your pin work!

Comment (hectorsector) [inline _pins/rohit-rohitdanda.json]: You'll also want to rename this file to be `_pins/rohitdanda.json`

**Comment by hollenberry** [comment]
Hey @huizhuwang ! As Hector noted above, if you make the changes he requested you'll be all set!

Please let us know if you need any help! We'll be closing this out to keep the repository clean if there's no action or conversation within the week.

**Comment by crichID** [comment]
@rohitdanda I went ahead and fixed the syntax issues in https://github.com/githubschool/open-enrollment-classes-introduction-to-github/pull/4782/commits/cef9d16f875f553d07f84f01026095d7faa87ee3. Please take a quick look at the changes. Once the checks have finished, please feel free to click the big green **Merge pull request** button. If you haven't merged it by April 21st, I will go ahead and do it just to keep the repository clean!

**Pull request merged**